src/Database/Esqueleto/Internal/Internal.hs

```diff
@@ -1007,12 +1007,13 @@ case_ = unsafeSqlCase
 --   Primary bar
 -- @
 --
--- For this example, declare:
+-- In this example, Bar is said to be the BaseEnt(ity), and Foo the child.
+-- To model this in Esqueleto, declare:
```

Comment:
Explicitly clarifying what is supposed to be the Base was helpful to me. It's kind of confusing b/c the code reads as `BaseEnt Foo`, which vaguely looks like Foo is the BaseEnt